collection_test.go

```diff
@@ -69,4 +70,87 @@ func TestCollectionScripts(t *testing.T) {
 		assert.JSONEq(otu.T, expected, result)
 
 	})
+
+	t.Run("Should be able to mint Dandy and then get it by script Index", func(t *testing.T) {
+
+		expected := `
```

Comment:
After talking to Christian, 

THis script needs to return 
 { "Dandy" : [81,82,80]} 
 
 So a dict of Collection names (from the registry) and an array of ids